.husky/pre-commit

```diff
@@ -1,13 +1,11 @@
 #!/bin/sh
 
-. "$(dirname "$0")/_/husky.sh"
-
 BRANCH=$(git rev-parse --abbrev-ref HEAD)
-REGEX="^(master|develop|((feat|fix|docs|chore|refactor|release|test)\/.+))$"
+REGEX="^(master|develop|((feat|fix|docs|chore|refactor|release|test|hotfix)\/.+))$"
```

Comment:
made this exactly the same as the REGEX in frontend-next, thought its better have it consistent